src/main/java/ay2021s1_cs2103_w16_3/finesse/model/frequent/FrequentExpense.java

```diff
@@ -0,0 +1,46 @@
+package ay2021s1_cs2103_w16_3.finesse.model.frequent;
+
+import java.util.Set;
+
+import ay2021s1_cs2103_w16_3.finesse.model.category.Category;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Amount;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Date;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Expense;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Title;
+
+public class FrequentExpense extends Expense {
```

Comment:
Should `FrequentExpense` be a subtype of `Expense`? Consider the fact that they appear in different lists, and are treated very differently. Moreover a method to convert a `FrequentExpense` to an `Expense` is implemented .Also note that `FrequentExpense` should not have a `Date` field, since its date is undefined. Only the `Expense` that it converts to should have a date.